### PR TITLE
Fix for Terraform Plan unit test - GitHub action

### PR DESCRIPTION
**Pull request by guptamukul-google**
Fix for #95

### CODE REVIEW

File: tools/check-tf-plan.py [hunk 1]
```diff
@@ -173,7 +180,11 @@ def tf(dir):
...
-  return_code_plan, stdout_plan, stderr_plan = tr.plan_cmd(capture_output=False,var={'billing_account_id':'ABCD-EFGH-IJKL-MNOP', 'organization_id':'1234567890', 'random_id': '1234'})
+
+  if "secure_data_warehouse" in dir:
+    return_code_plan, stdout_plan, stderr_plan = tr.plan_cmd(capture_output=False,var={'billing_account_id':'ABCD-EFGH-IJKL-MNOP', 'organization_id':'1234567890', 'random_id': '1234', 'data_analyst_group': 'data_analyst_group@example.com', 'data_engineer_group': 'data_engineer_group@example.com', 'security_administrator_group': 'security_administrator_group@example.com', 'network_administrator_group': 'network_administrator_group@example.com', 'security_analyst_group': 'security_analyst_group@example.com', 'perimeter_additional_members': ['demouser@example.com','demosa@service.gserviceaccount.com'], 'secure_datawarehouse_service_acccount': 'radlab-module-creator-sa@project-id.iam.gserviceaccount.com'})
+  else:
+    return_code_plan, stdout_plan, stderr_plan = tr.plan_cmd(capture_output=False,var={'billing_account_id':'ABCD-EFGH-IJKL-MNOP', 'organization_id':'1234567890', 'random_id': '1234'})
```
Comment (acolver) [inline tools/check-tf-plan.py:187]: Format this (e.g. `PEP8`) and add a comment on why this condition exists

File: modules/silicon/variables.tf [hunk 2]
```diff
@@ -66,20 +76,11 @@ variable "billing_budget_notification_email_addresses" {
 }
 
 variable "billing_budget_services" {
-  description = "A list of services ids to be included in the budget. If omitted, all services will be included in the budget. Service ids can be found at https://cloud.google.com/skus/. {{UIMeta group=0 order=13 updatesafe }}"
+  description = "A list of services ids to be included in the budget. If omitted, all services will be included in the budget. Service ids can be found at https://cloud.google.com/skus/. {{UIMeta group=0 order=14 updatesafe }}"
```
Comment (acolver) [inline modules/silicon/variables.tf:79]: Just FYI, there can be gaps in numbers. What matters to RLUI is the **relative** ordering (for example, `[1,2,3]` would order the same as `[1, 5, 9999]`). So if you delete a variable you don't need to update the others.

**Review by acolver — DISMISSED** [review]
A few minor comments, but overall LGTM

File: tools/check-tf-plan.py (hunk 1 above)
Comment (guptamukul-google) [reply tools/check-tf-plan.py]: Made the update

File: modules/silicon/variables.tf (hunk 2 above)
Comment (guptamukul-google) [reply modules/silicon/variables.tf:79]: Sounds good!

**Pull request merged**